modules/checklist/steps-manager.php

```diff
@@ -15,14 +15,17 @@
 class Steps_Manager {
 	/** @var Step_Base[] $step_instances */
 	private array $step_instances = [];
-	private array $steps_config = [];
 	private static array $step_ids = [ Add_Logo::STEP_ID, Create_Pages::STEP_ID, Setup_Header::STEP_ID, Assign_Homepage::STEP_ID ];
 
 	private Checklist_Module_Interface $module;
 
 	public function __construct( Checklist_Module_Interface $module ) {
 		$this->module = $module;
 		$this->register_steps();
+
+		add_action( 'elementor/init', function() {
```

Comment:
Why do we need to wrap this in `elementor/init`?

Reply:
The `add_filter` fires after the `apply_filters` ran, so it didn't "listen" to it

Maybe another hook?

Reply:
@ronenelementor Maybe you need to use `do_action` to allow developers to register more steps.
Check Display Conditions:
`do_action( 'elementor/display_conditions/register', $this );`

Reply:
I created a meeting for us tomorrow morning if we need to talk about this further :)